dev-qt/libqt5pas/libqt5pas-1.2.9.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="Pascal wrapper for Qt5"
+DESCRIPTION="libqt5pas is an interface between Lazarus and the Qt5 libraries.
+Not all Qt5 functions are available, only those necessary for Lazarus \
+functionality."
+HOMEPAGE="https://wiki.freepascal.org/Qt5_Interface"
+COPYRIGHT="2005-2013 by Jan Van hijfte"
```

Comment:
Not sure here, I see Jan as the original maintainer for libqt4pas, but I think Željan should be added as the current one (can't find it atm)

Reply:
Found it :)